docusaurus/docs/publish-a-plugin/sign-a-plugin.md

```diff
@@ -15,15 +15,30 @@ keywords:
 
 All Grafana Labs-authored backend plugins, including Enterprise plugins, are signed so that we can verify their authenticity with [signature verification](https://grafana.com/docs/grafana/latest/administration/plugin-management#plugin-signatures). By [default](https://grafana.com/docs/grafana/latest/administration/plugin-management#allow-unsigned-plugins), Grafana requires all plugins to be signed in order for them to be loaded.
 
-Before you can sign your plugin, you need to decide whether you want to sign it as a _public_ or a _private_ plugin.
+:::info
+
+It's not necessary to sign a plugin during development. The [Docker development environment](../get-started/set-up-development-environment.mdx) that is scaffolded with `@grafana/create-plugin` will load the plugin without a signature.
+
+:::
+
+## Public or private plugins
```

Comment:
I think we could probably tidy this section up a little, we don't need the multiple links to signature level, and in fact I think we could omit that entire section as we're better off linking to the full guidance - users will have to read that anyway. I think we also shouldn't link directly to the sign instructions for either type as users may skip the generation of an access policy token

This would be my suggestion, but also happy to discuss more or for alternatives

> Plugins can have different [signature levels](https://grafana.com/legal/plugins/#what-are-the-different-classifications-of-plugins) according to their author, related technology, and intended use. 
>
> The plugin can be _public_, signed as Community or Commercial, and distributed within the [Grafana plugins catalog](https://grafana.com/grafana/plugins/), so that it is available for others to install.  Or _private_ and only available for use within your organization.
>
> Before signing your plugin, review the [Plugins policy](https://grafana.com/legal/plugins/) to determine the appropriate signature for your plugin.

Reply:
done.

Reply:
The signature levels section is still there - I still think we should remove it to avoid repetition and the risk of folks missing the access policy step

Reply:
I'm not sure why you were seeing the sign levels section, it was removed. Please check again.

Reply:
It's still there
<img width="910" alt="Screenshot 2023-10-26 at 20 49 55" src="https://github.com/grafana/plugin-tools/assets/19860021/b1ffeaa9-0e97-4468-86c8-a6c37115792e">

Reply:
oh, THAT signature levels section. I thought you were talking about the one identical to https://grafana.com/legal/plugins/#what-are-the-different-classifications-of-plugins which was removed. done, thanks.